### PR TITLE
hw-1.1

### PR DESCRIPTION
**Pull request by Nick97-git**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return result1 == bet1 && result2 == bet2 ? 2 :
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: Good work) '?' should be on a new line

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
...
+                (result1 - result2 > 0 && bet1 - bet2 > 0)
+                        || (result1 - result2 < 0 && bet1 - bet2 < 0)
+                        || (result1 - result2 == 0 && bet1 - bet2 == 0) ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: same

Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
                (result1 - result2 > 0 && bet1 - bet2 > 0)
             || (result1 - result2 < 0 && bet1 - bet2 < 0)
             || (result1 - result2 == 0 && bet1 - bet2 == 0) 
             ? 1 : 0;
```

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 1 above)
Comment (Nick97-git) [reply src/main/java/core/basesyntax/FootballMatch.java]: Снимок1
Is it right?

Comment (Nick97-git) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**